recipes/python-chromedriver-binary/download-chromedriver.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+import subprocess
+import zipfile
+
+pkg_version = os.environ['PKG_VERSION']
+driver_version = "{0}.{1}".format(*pkg_version.split("."))
+if sys.platform == 'win32':
+    chromedriver_pkg = 'chromedriver_win32.zip'
+elif sys.platform == 'darwin':
+    chromedriver_pkg = 'chromedriver_mac64.zip'
+else:
+    chromedriver_pkg = 'chromedriver_linux64.zip'
+
+download_url = 'https://chromedriver.storage.googleapis.com/{0}/{1}'.format(driver_version, chromedriver_pkg)
```

Comment:
I think that latest conda-build supports multiple sources and would simplify this a lot while also getting the checksum checks for free. I'll merge this as is to get it into the channel and look into it later.

Reply:
Thanks @ocefpaf, I was not aware of that feature.

Do you know if there's documentation for it? All I could find was https://github.com/conda/conda-build/issues/1466 and https://github.com/conda/conda-build/pull/1929.

Reply:
https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-from-multiple-sources

Reply:
Thanks @isuruf!

Reply:
The real mystery is what @isuruf does not know :wink:

Reply:
Thks guys. I'll change the recipe